app/index.html

```diff
@@ -81,11 +81,16 @@
             </h1>
           </div>
 
+          <div class="card-pf-heading">
+            <h2 class="card-pf-title">
+              Recommender
+            </h2>
+          </div>
```

Comment:
@pmuir wanted to have common heading "Recommender" for below two cards i.e "analysis of component" and "analysis of Stack" . Shall i make use of tab panel or something else?